apps/web/src/hooks/useRoutes.ts

```diff
@@ -304,6 +304,9 @@ const getRoutes = (lang: Locales) => {
         "digital-services-contracting-questionnaire",
       ),
 
+    // Applicant Settings
+    applicantSettings: () => path.join(applicantUrl, "settings"),
```

Comment:
Shouldn't this be called Account settings since it is not just for applicants but the user's account generally?

## Screenshot
<img width="1200" alt="Screen Shot 2024-04-05 at 09 59 54" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/637ae2d0-7e76-411d-bf46-a56abbe07fb4">

Reply:
Good point 👍 Fixed by [0925f40](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/0925f4080b5e0b990111afa5bad57669af83dae9)